cbor.js

```diff
@@ -172,7 +260,7 @@ function encode(value) {
   }
 
   encodeItem(value);
-
+  /* istanbul ignore if */
   if ("slice" in data)
     return data.slice(0, offset);
```

Comment:
this branch is using a feature test that never succeeds in coverage testing, and should either be ignored by coverage or removed